x-pack/test/saved_object_tagging/api_integration/security_and_spaces/apis/_get_assignable_types.ts

```diff
@@ -32,7 +32,7 @@ export default function (ftrContext: FtrProviderContext) {
     });
 
     const assignablePerUser = {
-      [USERS.SUPERUSER.username]: ['dashboard', 'visualization', 'map', 'lens'],
+      [USERS.SUPERUSER.username]: ['dashboard', 'visualization', 'map', 'lens', 'search'],
```

Comment:
I also wasn't sure about this.

Reply:
Adding it was correct, this test checks the assignable types per users, and the SU sees all types.